### PR TITLE
Deploy website docs using Github Action

### PR DESCRIPTION
**Pull request by jrodbx**

### CODE REVIEW

File: RELEASING.md [hunk 1]
```diff
@@ -11,8 +11,7 @@ Releasing
  8. Update the `gradle.properties` to the next SNAPSHOT version.
  9. `git commit -am "Prepare next development version"`
  10. `git push && git push --tags`
- 11. `./deploy_website.sh`
```
Comment (JakeWharton) [inline RELEASING.md:14]: Delete the script?

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -66,3 +66,11 @@ jobs:
         env:
           ORG_GRADLE_PROJECT_SONATYPE_NEXUS_USERNAME: ${{ secrets.SONATYPE_NEXUS_USERNAME }}
           ORG_GRADLE_PROJECT_SONATYPE_NEXUS_PASSWORD: ${{ secrets.SONATYPE_NEXUS_PASSWORD }}
+
+      - name: Deploy docs to website
+        uses: JamesIves/github-pages-deploy-action@releases/v3
+        with:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          BRANCH: gh-pages
+          FOLDER: build/dokka/html
```
Comment (JakeWharton) [inline .github/workflows/build.yml:75]: I suppose `uploadArchives` creates this folder?

**Review by JakeWharton — APPROVED** [review]

File: .github/workflows/build.yml (hunk 2 above)
Comment (jrodbx) [reply .github/workflows/build.yml:75]: hmm, doesn't look like it.  I just cribbed this from `https://github.com/cashapp/turbine/blob/trunk/.github/workflows/build.yaml#L50`

Comment (jrodbx) [reply .github/workflows/build.yml:75]: Ah, my guess is that, in build.yaml, either `build` or `publish` (prob the former) executes the dokka task which creates that folder?

Comment (jrodbx) [reply .github/workflows/build.yml:75]: And in release.yaml, `dokkaHtml` does it?  https://github.com/cashapp/turbine/blob/trunk/.github/workflows/release.yaml#L64-L65

Comment (jrodbx) [reply .github/workflows/build.yml:75]: hmm, i should also update the dokka plugin...

Comment (jrodbx) [reply .github/workflows/build.yml]: Ok, had to re-learn the dokka config 😅 

This repo doesn't use the html style that Turbine uses, but rather the gfm style that SqlDelight et al. use.

**Pull request merged**